gramps/gen/lib/test/datadict_test.py

```diff
@@ -0,0 +1,55 @@
+#
+# Gramps - a GTK+/GNOME based genealogy program
+#
+# Copyright (C) 2025      Doug Blank
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+""" Unittest for DataDict API """
+
+# See also tests in ./serialize_test.py
+
+import unittest
+
+from gramps.gen.lib.serialize import DataDict, to_dict, from_dict
+from gramps.gen.lib import (
+    Person,
+    Family,
+)
+
+
+class DataDictTest(unittest.TestCase):
```

Comment:
Add unittests for DataList as well?

Reply:
`DataList` is checked implicitly inside the `DataDict` tests. But, I can add some specific one too.

Reply:
Done!